src/FacadeAccountCreation/FacadeAccountCreation.API/Controllers/PersonsController.cs

```diff
@@ -74,4 +79,35 @@ public async Task<IActionResult> GetPersonFromInviteToken([FromQuery] string tok
 
         return personResponse == null ? NotFound() : Ok(personResponse);
     }
+
+    [HttpPut]
+    [Route("update-user-details")]
+    [Consumes("application/json")]
+    [ProducesResponseType(StatusCodes.Status200OK)]
+    [ProducesResponseType(StatusCodes.Status400BadRequest)]
+    [ProducesResponseType(StatusCodes.Status500InternalServerError)]
+    public async Task<IActionResult> PutUserDetailsByUserId(
+        Guid userId,    
+        [FromBody] UserDetailsDto userDetailsDto)
+    {
+        if (userId == null)
```

Comment:
I have a sneaky suspicions that the user id does not need passing across: it's part of the authentication process, so I believe it is available to you from the controller without it being a parameters